src/NextGen/DonationForm/Controllers/DonationFormViewController.php

```diff
@@ -32,7 +33,10 @@ public function show(DonationFormViewRouteData $data): string
         $viewModel = new DonationFormViewModel($data->formId, $data->formBlocks);
 
         wp_enqueue_global_styles();
-        $this->enqueueFormScripts($data->formId, $data->formTemplateId);
+        $this->enqueueFormScripts($data->formId, $data->formSettings['templateId']);
+
+        $primaryColor = $data->formSettings['primaryColor'];
+        $secondaryColor = $data->formSettings['secondaryColor'];
```

Comment:
Let's add a nullish condition for these to display a string if null. I like having the flexibility of piecemealing the form settings when it comes to the iframe url arguments.